chef/rsa.py

```diff
@@ -8,28 +9,30 @@
 else:
     _eay = CDLL('libcrypto.so')
 
-#unsigned long ERR_get_error(void);
+# unsigned long ERR_get_error(void);
```

Comment:
Mentioned above, but please revert what I assume was running flake8 on this. That can be its own PR.